src/components/NegationToggle.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<span class="negationToggle">
+		<Button
+			:type="value==='with' ? 'progressive' : 'neutral'"
```

Comment:
it doesn't seem to change to progressive (as in blue background). when it is selected it only shows a blue border around the selected button. is this the desired effect?